sass/styles/chart.scss

```diff
@@ -975,3 +975,39 @@ html, body {
 .ps--active-y > .ps__rail-y {
     background-color: transparent !important;
 }
+
+$spot-color: #0298d3;
+
+.cq-spot {
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    cursor: pointer;
+    position: absolute;
+    background: $spot-color;
+    @include box-shadow-raw(0 0 0 rgba($spot-color, 0.6));
+    animation: pulse 1.6s infinite;
+}
+
+@-webkit-keyframes pulse {
```

Comment:
Can you write a mixin for keyframes (or just find one online)? You have `@-webkit-keyframes` and `@keyframes` having exactly the same code

Reply:
oops, you are right, sorry I don't get your point on the first time.